go_utils/photo_download.py

```diff
@@ -325,6 +429,8 @@ def download_lc_photos(
     south_photo="lc_SouthPhotoUrl",
     east_photo="lc_EastPhotoUrl",
     west_photo="lc_WestPhotoUrl",
+    include_in_name=[],
+    additional_name_stem="",
```

Comment:
See additional name comment in the tests

Reply:
see https://github.com/IGES-Geospatial/globe-observer-utils/pull/73#discussion_r776763168